server/sttp-mock-server/src/test/scala/sttp/tapir/server/mockserver/SttpMockServerClientSpec.scala

```diff
@@ -190,5 +190,33 @@ class SttpMockServerClientSpec extends AnyFlatSpec with Matchers with BeforeAndA
     actual shouldEqual Success(Value(Right(sampleOut)))
   }
 
+  it should "match json null body" in {
+    import TapirJsonCirceWithDropNullEnabled.jsonBody
+
+    val orderEndpoint = endpoint
+      .in("api" / "v1" / "order")
+      .get
+      .errorOut(jsonBody[ApiError])
+      .out(jsonBody[OrderCreatedEvent])
+
+    val sampleOut = OrderCreatedEvent(id = uuid(), name = "John", total = None)
+
+    val actual = for {
+      _ <- mockServerClient
+        .whenInputMatches(orderEndpoint)((), None)
```

Comment:
shouldn't this `None` be a `()`? There are compilation warnings with this form

Reply:
Good catch, I didn't pay attention to that warning :+1:

Reply:
48fb6e6